ArduPlane/defines.h

```diff
@@ -153,3 +154,9 @@ enum CrowFlapOptions {
     PROGRESSIVE_CROW = (1 << 2),
 }; 
 
+
+enum guided_heading_type_t {
```

Comment:
Should be `enum class`

Reply:
this enum as-is is consistent with all the other 'enum'-s in defines.h so i wonder why you suggest making it different to the rest ?